apps/nextjs-website/src/app/[productSlug]/tutorials/[...productTutorialPage]/page.tsx

```diff
@@ -76,168 +44,37 @@ export async function generateMetadata({
       url: path,
     });
   }
-  const {
-    page: { path, title },
-  } = await getStaticTutorial(productSlug, [tutorialPath]);
-
-  return makeMetadata({
-    title,
-    url: path,
-  });
 }
 
 const Page = async ({ params }: { params: Params }) => {
   const productSlug = params?.productSlug;
   const tutorialPath = params?.productTutorialPage?.join('/');
 
-  const strapiTutorialProps = await getStrapiTutorial(productSlug, [
-    tutorialPath,
-  ]);
-
-  if (strapiTutorialProps) {
-    const structuredData = generateStructuredDataScripts({
-      breadcrumbsItems: [
-        productToBreadcrumb(strapiTutorialProps.product),
-        {
-          name: strapiTutorialProps.seo?.metaTitle,
-          item: breadcrumbItemByProduct(strapiTutorialProps.product, [
-            'guides',
-            ...(params?.productTutorialPage || []),
-          ]),
-        },
-      ],
-      seo: strapiTutorialProps.seo,
-    });
-    return (
-      <TutorialTemplate
-        bannerLinks={strapiTutorialProps.bannerLinks}
-        parts={strapiTutorialProps.parts}
-        path={strapiTutorialProps.path}
-        product={strapiTutorialProps.product}
-        relatedLinks={strapiTutorialProps.relatedLinks}
-        title={strapiTutorialProps.title}
-        structuredData={structuredData}
-      />
-    );
-  }
-
-  const urlReplaceMap = await getUrlReplaceMapProps();
-  const tutorialProps = await getStaticTutorial(productSlug, [tutorialPath]);
-  const { product, page, bannerLinks, source, relatedLinks } = tutorialProps;
-  const props: ProductTutorialPageProps = {
-    ...page,
-    product,
-    bannerLinks,
-    relatedLinks,
-    bodyConfig: {
-      isPageIndex: false,
-      pagePath: page.path,
-      assetsPrefix: source.assetsPrefix,
-      gitBookPagesWithTitle,
-      spaceToPrefix: spaceToPrefixMap,
-      urlReplaces: urlReplaceMap,
-    },
-  };
-
-  const hasRelatedLinks = (props.relatedLinks?.links?.length ?? 0) > 0;
+  const strapiTutorialProps = await getTutorial(productSlug, [tutorialPath]);
 
   const structuredData = generateStructuredDataScripts({
     breadcrumbsItems: [
-      productToBreadcrumb(product),
+      productToBreadcrumb(strapiTutorialProps.product),
       {
-        name: tutorialProps.page.title,
-        item: breadcrumbItemByProduct(product, [
+        name: strapiTutorialProps.seo?.metaTitle,
+        item: breadcrumbItemByProduct(strapiTutorialProps.product, [
           'guides',
           ...(params?.productTutorialPage || []),
         ]),
       },
     ],
-    seo: undefined,
+    seo: strapiTutorialProps.seo,
   });
-
   return (
-    <ProductLayout
-      product={props.product}
-      path={props.path}
-      bannerLinks={props.bannerLinks}
+    <TutorialTemplate
```

Comment:
Take a look at the breadcrumbs
![image](https://github.com/user-attachments/assets/49b27cae-530d-4cb2-ae28-04c056c9c1df)
![image](https://github.com/user-attachments/assets/51c41e30-a5f2-4e60-a11b-5cf3b23527f6)

https://dev.developer.pagopa.it/app-io/tutorials/come-utilizzare-correttamente-servizi-e-messaggi-in-app-io